### PR TITLE
accessibility-label rule does not require an accessibilityLabel when accessible={false}

### PR DESCRIPTION
**Pull request by mattvot**
Like #43, do not require an accessibilityLabel when accessible is `false`

### CODE REVIEW

**Comment by MoOx** [comment]
@mattvot Could you merge conflict?

File: src/rules/accessibility-label.js [hunk 1]
```diff
@@ -34,31 +34,30 @@ module.exports = {
...
-    }
-  })
+	create: (context: ESLintContext) => ({
+		JSXOpeningElement: (node: JSXOpeningElement) => {
+			const accessible = getProp(node.attributes, 'accessible');
+			if (!accessible || (accessible.value && accessible.value.expression.value === false)) return;
+			const labelPropVal = getAccessibilityLabel(node);
+			if (!labelPropVal) {
+				let childWithLabel;
+				if (node.parent) {
```
Comment (alex-saunders) [inline src/rules/accessibility-label.js]: Flow is failing here as it does not recognise `parent`, can we add a `$FlowFixMe` please? I think this was added in a recent commit but has maybe been overridden accidentally?

**Comment by MoOx** [comment]
@mattvot do you need any help?

**Comment by mattvot** [comment]
I'll try give it a go next week. I've been away for the last few weeks. Thanks.

**Comment by mattvot** [comment]
If someone wants to update the PR or add separately before I can update myself please go ahead 🙂

File: src/rules/accessibility-label.js (hunk 1 above)
Comment (mattvot) [reply src/rules/accessibility-label.js]: Added

**Comment by mattvot** [comment]
Ok, updated now. Let me know if anymore changes are required. Thanks

**Comment by mebobby2** [comment]
Any updates on when this will be merged?

**Comment by alex-saunders** [comment]
Sorry for the delay, thanks for your work on this @mattvot ! Merging now.

**Comment by mebobby2** [comment]
@alex-saunders Thanks for merging this in. Any chance of making a new release for this?

**Pull request merged**